### DISCUSSION TITLE
Database access GUI for Redshift

### DISCUSSION THREAD

**Discussion by leandrozimmer** [gravitational/teleport#12926] [Q&A] [answered]
Hi, 

Is possible connect JDBC ou ODBC redshift with GUI?

https://goteleport.com/docs/database-access/guides/gui-clients/

Thanks,
Zimmer

**Comment by webvictim** [comment, marked as answer]
I believe this should work, yes. We don't have a guide, but the steps should pretty much be the same as for any other GUI client - just with a different connector type.

**Reply by taraspos** [reply]
We have a guide now!
- https://goteleport.com/docs/connect-your-client/third-party/gui-clients/#redshift-dbeaver
